packages/shared_preferences/shared_preferences_platform_interface/lib/method_channel_shared_preferences.dart

```diff
@@ -45,12 +45,12 @@ class MethodChannelSharedPreferencesStore
   }
 
   @override
-  Future<bool> clear() {
+  Future<bool?> clear() {
     return _kChannel.invokeMethod<bool>('clear');
   }
 
   @override
-  Future<Map<String, Object>> getAll() {
+  Future<Map<String, Object>?> getAll() {
```

Comment:
The return type of `invokeMapMethod` is `Future<Map<K, V>?>`, that is why the return type of `clear` is `Future<Map<String, Object>?>`.